doc/stdenv/stdenv.chapter.md

```diff
@@ -4,7 +4,92 @@ The standard build environment in the Nix Packages collection provides an enviro
 
 ## Using `stdenv` {#sec-using-stdenv}
 
-To build a package with the standard environment, you use the function `stdenv.mkDerivation`, instead of the primitive built-in function `derivation`, e.g.
+To build a package with the standard environment, use the function [`stdenv.mkDerivation`](https://github.com/NixOS/nixpkgs/blob/ffa4e2da136d0aea78f1fc0e1ff8a646edbd6af5/pkgs/stdenv/generic/make-derivation.nix#L71-L166).
+
+### Parameters
+
+{
+
+  depsBuildBuild                    ? [] # -1 -> -1
+, depsBuildBuildPropagated          ? [] # -1 -> -1
+, nativeBuildInputs                 ? [] # -1 ->  0  N.B. Legacy name
+, propagatedNativeBuildInputs       ? [] # -1 ->  0  N.B. Legacy name
+, depsBuildTarget                   ? [] # -1 ->  1
+, depsBuildTargetPropagated         ? [] # -1 ->  1
+
+, depsHostHost                      ? [] #  0 ->  0
+, depsHostHostPropagated            ? [] #  0 ->  0
+, buildInputs                       ? [] #  0 ->  1  N.B. Legacy name
+, propagatedBuildInputs             ? [] #  0 ->  1  N.B. Legacy name
+
+, depsTargetTarget                  ? [] #  1 ->  1
+, depsTargetTargetPropagated        ? [] #  1 ->  1
+
+, checkInputs                       ? []
+, installCheckInputs                ? []
+, nativeCheckInputs                 ? []
+, nativeInstallCheckInputs          ? []
+
+# Configure Phase
+, configureFlags ? []
+, cmakeFlags ? []
```

Comment:
What??

Reply:
This is just copypasta from `make-derivation.nix` for now.

Reply:
Many of these vars should not be here, but in the setup hooks...